pom.xml

```diff
@@ -133,6 +133,8 @@
     <version.maven-source-plugin>2.4</version.maven-source-plugin>
     <version.maven-surefire-plugin>2.17</version.maven-surefire-plugin>
     <version.maven-war-plugin>2.5</version.maven-war-plugin>
+    <version.org.codehaus.gmavenplus.gmavenplus-plugin>1.2</version.org.codehaus.gmavenplus.gmavenplus-plugin>
+    <version.org.bsc.maven.maven-processor-plugin>2.0.2</version.org.bsc.maven.maven-processor-plugin>
```

Comment:
I have  a very trivial comment to make - if we want this list in alphabetical order, "org.codehaus..." should come after "org.bsc..." - so switch lines 136 and 137.